zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/DataStoreManager.scala

```diff
@@ -142,7 +142,7 @@ class DataStoreManager(metaDataset: String,
     val now = DateTime.now()
     val fixEndFilter = FilterStatement(sourceInfo.schema.timeField, None, Relation.<, Seq(TimeField.TimeFormat.print(now)))
     val newCreateQuery = create.query.copy(filter = fixEndFilter +: create.query.filter)
-    val queryString = managerParser.generate(create.copy(query = newCreateQuery), Map(newCreateQuery.dataset -> sourceInfo.schema))
+    val queryString = managerParser.generate(create.copy(query = newCreateQuery), Map(create.dataset -> sourceInfo.schema))
```

Comment:
I think this should be the `create.query.dataset` that contains the sourceInformation. `create.dataset` is the view dataset that needs to be created. 

We may need to rename it to clearly distinguish two different dataset. Maybe rename the `create.dataset` to `create.targetDataset`?